samtranslator/region_configuration.py

```diff
@@ -18,7 +18,14 @@ def is_apigw_edge_configuration_supported(cls) -> bool:
         :return: True, if API Gateway does not support Edge configuration
         """
 
-        return ArnGenerator.get_partition_name() not in ["aws-us-gov", "aws-iso", "aws-iso-b", "aws-cn", "aws-iso-e"]
+        return ArnGenerator.get_partition_name() not in [
+            "aws-us-gov",
+            "aws-iso",
+            "aws-iso-b",
+            "aws-cn",
+            "aws-iso-e",
```

Comment:
Should we just check for `aws-iso-*`?

Reply:
good point, added check to return false for all regions that start with aws-iso